broadcaster/sequencenumbercatchupbuffer.go

```diff
@@ -174,9 +174,3 @@ func (b *SequenceNumberCatchupBuffer) OnDoBroadcast(bmi interface{}) error {
 func (b *SequenceNumberCatchupBuffer) GetMessageCount() int {
 	return int(atomic.LoadInt32(&b.messageCount))
 }
-
-// Not thread safe
-func (b *SequenceNumberCatchupBuffer) Reset(messages []*BroadcastFeedMessage) {
-	b.messages = messages
-	atomic.StoreInt32(&b.messageCount, int32(len(messages)))
-}
```

Comment:
Nice, I like that you were able to remove this. Now there's only one way for messages to get into the catchup buffer.